src/bh_db_worker.erl

```diff
@@ -91,7 +47,7 @@ prepared_query(Pool, Name, Params) ->
                 {_Conn, Ref, Res} ->
                     Res
             after
-                500 ->
+                ?POOL_QUERY_TIMEOUT ->
```

Comment:
all queries should return within 5 seconds, or would it be better to take a timeout parameter if some queries are slower?

Reply:
I'd hope that's a serious upper bound. We can add a timeout parameter and expose another arity when we need it